<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "https://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="https://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
<title>TestNG Report</title>
<style type="text/css">table {margin-bottom:10px;border-collapse:collapse;empty-cells:show}th,td {border:1px solid #009;padding:.25em .5em}th {vertical-align:bottom}td {vertical-align:top}table a {font-weight:bold}.stripe td {background-color: #E6EBF9}.num {text-align:right}.passedodd td {background-color: #3F3}.passedeven td {background-color: #0A0}.skippedodd td {background-color: #DDD}.skippedeven td {background-color: #CCC}.failedodd td,.attn {background-color: #F33}.failedeven td,.stripe .attn {background-color: #D00}.stacktrace {white-space:pre;font-family:monospace}.totop {font-size:85%;text-align:center;border-bottom:2px solid #000}.invisible {display:none}</style>
</head>
<body>
<table>
<tr><th>Test</th><th># Passed</th><th># Skipped</th><th># Retried</th><th># Failed</th><th>Time (ms)</th><th>Included Groups</th><th>Excluded Groups</th></tr>
<tr><th colspan="8">Mobile Testing Project</th></tr>
<tr><td><a href="#t0">DolingoMobile Testing Project</a></td><td class="num">0</td><td class="num attn">10</td><td class="num">0</td><td class="num">0</td><td class="num">27,652</td><td></td><td></td></tr>
</table>
<table id='summary'><thead><tr><th>Class</th><th>Method</th><th>Start</th><th>Time (ms)</th></tr></thead><tbody><tr><th colspan="4">Mobile Testing Project</th></tr></tbody><tbody id="t0"><tr><th colspan="4">DolingoMobile Testing Project &#8212; failed (configuration methods)</th></tr><tr class="failedeven"><td rowspan="1">Badawi_Test.TS3_DeleteTask</td><td><a href="#m0">setUp</a></td><td rowspan="1">1759857832117</td><td rowspan="1">20590</td></tr><tr class="failedodd"><td rowspan="1">Badawi_Test.TS4_CompleteTask</td><td><a href="#m1">setUp</a></td><td rowspan="1">1759857852774</td><td rowspan="1">6964</td></tr><tr><th colspan="4">DolingoMobile Testing Project &#8212; skipped (configuration methods)</th></tr><tr class="skippedeven"><td rowspan="1">Badawi_Test.TS3_DeleteTask</td><td><a href="#m2">tearDown</a></td><td rowspan="1">1759857852772</td><td rowspan="1">0</td></tr><tr class="skippedodd"><td rowspan="1">Badawi_Test.TS4_CompleteTask</td><td><a href="#m3">tearDown</a></td><td rowspan="1">1759857859746</td><td rowspan="1">0</td></tr><tr><th colspan="4">DolingoMobile Testing Project &#8212; skipped</th></tr><tr class="skippedeven"><td rowspan="9">Badawi_Test.TS3_DeleteTask</td><td><a href="#m4">AddDemoTasks</a></td><td rowspan="1">1759857852730</td><td rowspan="1">0</td></tr><tr class="skippedeven"><td><a href="#m5">CancelDeleteTask</a></td><td rowspan="1">1759857852743</td><td rowspan="1">0</td></tr><tr class="skippedeven"><td><a href="#m6">ClearCompleteTask</a></td><td rowspan="1">1759857852763</td><td rowspan="1">0</td></tr><tr class="skippedeven"><td><a href="#m7">DeleteAllTask</a></td><td rowspan="1">1759857852758</td><td rowspan="1">0</td></tr><tr class="skippedeven"><td><a href="#m8">DeleteFromSearch</a></td><td rowspan="1">1759857852769</td><td rowspan="1">0</td></tr><tr class="skippedeven"><td><a href="#m9">DeleteFromTask</a></td><td rowspan="1">1759857852748</td><td rowspan="1">0</td></tr><tr class="skippedeven"><td><a href="#m10">DeleteMixedTask</a></td><td rowspan="1">1759857852766</td><td rowspan="1">0</td></tr><tr class="skippedeven"><td><a href="#m11">DeleteMultipleTasks</a></td><td rowspan="1">1759857852752</td><td rowspan="1">1</td></tr><tr class="skippedeven"><td><a href="#m12">DeleteTask</a></td><td rowspan="1">1759857852737</td><td rowspan="1">0</td></tr><tr class="skippedodd"><td rowspan="1">Badawi_Test.TS4_CompleteTask</td><td><a href="#m13">testAddSimpleTask</a></td><td rowspan="1">1759857859741</td><td rowspan="1">0</td></tr></tbody>
</table>
<h2>DolingoMobile Testing Project</h2><h3 id="m0">Badawi_Test.TS3_DeleteTask#setUp</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:216)
	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:225)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:89)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:99)
	at io.appium.java_client.android.AndroidDriver.&lt;init&gt;(AndroidDriver.java:97)
	at base.TestBase.setUp(TestBase.java:40)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:193)
	... 38 more
Caused by: org.openqa.selenium.WebDriverException: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:215)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:1034)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:147)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:588)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:574)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:687)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	... 40 more
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m1">Badawi_Test.TS4_CompleteTask#setUp</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: Error executing adbExec. Original error: &apos;Command &apos;&apos;C:\Users\moham\AppData\Local\Android\Sdk\platform-tools\adb.exe&apos; -P 5037 -s emulator-5554 shell &apos;settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy&apos;&apos; exited with code 20&apos;; Command output: cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings

Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Error executing adbExec. Original error: &apos;Command &apos;&apos;C:\Users\moham\AppData\Local\Android\Sdk\platform-tools\adb.exe&apos; -P 5037 -s emulator-5554 shell &apos;settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy&apos;&apos; exited with code 20&apos;; Command output: cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings

    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Error executing adbExec. Original error: &apos;Command &apos;&apos;C:\Users\moham\AppData\Local\Android\Sdk\platform-tools\adb.exe&apos; -P 5037 -s emulator-5554 shell &apos;settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy&apos;&apos; exited with code 20&apos;; Command output: cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings

    at ChildProcess.&lt;anonymous&gt; (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\teen_process\lib\exec.js:110:19)
    at ChildProcess.emit (node:events:519:28)
    at maybeClose (node:internal/child_process:1101:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:304:5)
    at Process.callbackTrampoline (node:internal/async_hooks:130:17)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:216)
	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:225)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:89)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:99)
	at io.appium.java_client.android.AndroidDriver.&lt;init&gt;(AndroidDriver.java:97)
	at base.TestBase.setUp(TestBase.java:40)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:193)
	... 38 more
Caused by: org.openqa.selenium.WebDriverException: Error executing adbExec. Original error: &apos;Command &apos;&apos;C:\Users\moham\AppData\Local\Android\Sdk\platform-tools\adb.exe&apos; -P 5037 -s emulator-5554 shell &apos;settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy&apos;&apos; exited with code 20&apos;; Command output: cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings

Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Error executing adbExec. Original error: &apos;Command &apos;&apos;C:\Users\moham\AppData\Local\Android\Sdk\platform-tools\adb.exe&apos; -P 5037 -s emulator-5554 shell &apos;settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy&apos;&apos; exited with code 20&apos;; Command output: cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings

    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Error executing adbExec. Original error: &apos;Command &apos;&apos;C:\Users\moham\AppData\Local\Android\Sdk\platform-tools\adb.exe&apos; -P 5037 -s emulator-5554 shell &apos;settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy&apos;&apos; exited with code 20&apos;; Command output: cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings

    at ChildProcess.&lt;anonymous&gt; (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\teen_process\lib\exec.js:110:19)
    at ChildProcess.emit (node:events:519:28)
    at maybeClose (node:internal/child_process:1101:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:304:5)
    at Process.callbackTrampoline (node:internal/async_hooks:130:17)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:215)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:1034)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:147)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:588)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:574)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:687)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	... 40 more
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m2">Badawi_Test.TS3_DeleteTask#tearDown</h3><table class="result"><tr><th class="invisible"/></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m3">Badawi_Test.TS4_CompleteTask#tearDown</h3><table class="result"><tr><th class="invisible"/></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m4">Badawi_Test.TS3_DeleteTask#AddDemoTasks</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:216)
	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:225)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:89)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:99)
	at io.appium.java_client.android.AndroidDriver.&lt;init&gt;(AndroidDriver.java:97)
	at base.TestBase.setUp(TestBase.java:40)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:193)
	... 38 more
Caused by: org.openqa.selenium.WebDriverException: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:215)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:1034)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:147)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:588)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:574)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:687)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	... 40 more
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m5">Badawi_Test.TS3_DeleteTask#CancelDeleteTask</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:216)
	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:225)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:89)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:99)
	at io.appium.java_client.android.AndroidDriver.&lt;init&gt;(AndroidDriver.java:97)
	at base.TestBase.setUp(TestBase.java:40)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:193)
	... 38 more
Caused by: org.openqa.selenium.WebDriverException: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:215)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:1034)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:147)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:588)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:574)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:687)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	... 40 more
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m6">Badawi_Test.TS3_DeleteTask#ClearCompleteTask</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:216)
	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:225)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:89)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:99)
	at io.appium.java_client.android.AndroidDriver.&lt;init&gt;(AndroidDriver.java:97)
	at base.TestBase.setUp(TestBase.java:40)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:193)
	... 38 more
Caused by: org.openqa.selenium.WebDriverException: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:215)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:1034)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:147)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:588)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:574)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:687)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	... 40 more
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m7">Badawi_Test.TS3_DeleteTask#DeleteAllTask</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:216)
	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:225)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:89)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:99)
	at io.appium.java_client.android.AndroidDriver.&lt;init&gt;(AndroidDriver.java:97)
	at base.TestBase.setUp(TestBase.java:40)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:193)
	... 38 more
Caused by: org.openqa.selenium.WebDriverException: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:215)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:1034)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:147)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:588)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:574)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:687)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	... 40 more
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m8">Badawi_Test.TS3_DeleteTask#DeleteFromSearch</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:216)
	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:225)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:89)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:99)
	at io.appium.java_client.android.AndroidDriver.&lt;init&gt;(AndroidDriver.java:97)
	at base.TestBase.setUp(TestBase.java:40)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:193)
	... 38 more
Caused by: org.openqa.selenium.WebDriverException: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:215)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:1034)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:147)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:588)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:574)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:687)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	... 40 more
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m9">Badawi_Test.TS3_DeleteTask#DeleteFromTask</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:216)
	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:225)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:89)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:99)
	at io.appium.java_client.android.AndroidDriver.&lt;init&gt;(AndroidDriver.java:97)
	at base.TestBase.setUp(TestBase.java:40)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:193)
	... 38 more
Caused by: org.openqa.selenium.WebDriverException: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:215)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:1034)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:147)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:588)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:574)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:687)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	... 40 more
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m10">Badawi_Test.TS3_DeleteTask#DeleteMixedTask</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:216)
	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:225)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:89)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:99)
	at io.appium.java_client.android.AndroidDriver.&lt;init&gt;(AndroidDriver.java:97)
	at base.TestBase.setUp(TestBase.java:40)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:193)
	... 38 more
Caused by: org.openqa.selenium.WebDriverException: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:215)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:1034)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:147)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:588)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:574)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:687)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	... 40 more
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m11">Badawi_Test.TS3_DeleteTask#DeleteMultipleTasks</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:216)
	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:225)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:89)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:99)
	at io.appium.java_client.android.AndroidDriver.&lt;init&gt;(AndroidDriver.java:97)
	at base.TestBase.setUp(TestBase.java:40)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:193)
	... 38 more
Caused by: org.openqa.selenium.WebDriverException: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:215)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:1034)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:147)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:588)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:574)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:687)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	... 40 more
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m12">Badawi_Test.TS3_DeleteTask#DeleteTask</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:216)
	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:225)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:89)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:99)
	at io.appium.java_client.android.AndroidDriver.&lt;init&gt;(AndroidDriver.java:97)
	at base.TestBase.setUp(TestBase.java:40)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:193)
	... 38 more
Caused by: org.openqa.selenium.WebDriverException: Could not find a connected Android device in 20000ms
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Could not find a connected Android device in 20000ms
    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Could not find a connected Android device in 20000ms
    at ADB.getDevicesWithRetry (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-adb\lib\tools\system-calls.js:296:13)
    at AndroidUiautomator2Driver.getDeviceInfoFromCaps (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\appium-android-driver\lib\commands\device\common.js:41:21)
    at AndroidUiautomator2Driver.createSession (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\lib\driver.ts:361:30)
    at AppiumDriver.createSession (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\lib\appium.js:360:35)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:215)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:1034)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:147)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:588)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:574)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:687)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	... 40 more
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m13">Badawi_Test.TS4_CompleteTask#testAddSimpleTask</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: Error executing adbExec. Original error: &apos;Command &apos;&apos;C:\Users\moham\AppData\Local\Android\Sdk\platform-tools\adb.exe&apos; -P 5037 -s emulator-5554 shell &apos;settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy&apos;&apos; exited with code 20&apos;; Command output: cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings

Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Error executing adbExec. Original error: &apos;Command &apos;&apos;C:\Users\moham\AppData\Local\Android\Sdk\platform-tools\adb.exe&apos; -P 5037 -s emulator-5554 shell &apos;settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy&apos;&apos; exited with code 20&apos;; Command output: cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings

    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Error executing adbExec. Original error: &apos;Command &apos;&apos;C:\Users\moham\AppData\Local\Android\Sdk\platform-tools\adb.exe&apos; -P 5037 -s emulator-5554 shell &apos;settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy&apos;&apos; exited with code 20&apos;; Command output: cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings

    at ChildProcess.&lt;anonymous&gt; (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\teen_process\lib\exec.js:110:19)
    at ChildProcess.emit (node:events:519:28)
    at maybeClose (node:internal/child_process:1101:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:304:5)
    at Process.callbackTrampoline (node:internal/async_hooks:130:17)
Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:216)
	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:225)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.&lt;init&gt;(RemoteWebDriver.java:131)
	at io.appium.java_client.DefaultGenericMobileDriver.&lt;init&gt;(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:89)
	at io.appium.java_client.AppiumDriver.&lt;init&gt;(AppiumDriver.java:99)
	at io.appium.java_client.android.AndroidDriver.&lt;init&gt;(AndroidDriver.java:97)
	at base.TestBase.setUp(TestBase.java:40)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:193)
	... 38 more
Caused by: org.openqa.selenium.WebDriverException: Error executing adbExec. Original error: &apos;Command &apos;&apos;C:\Users\moham\AppData\Local\Android\Sdk\platform-tools\adb.exe&apos; -P 5037 -s emulator-5554 shell &apos;settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy&apos;&apos; exited with code 20&apos;; Command output: cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings

Build info: version: &apos;3.141.59&apos;, revision: &apos;e82be7d358&apos;, time: &apos;2018-11-14T08:17:03&apos;
System info: host: &apos;BADAWI&apos;, ip: &apos;192.168.1.137&apos;, os.name: &apos;Windows 11&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;24.0.2&apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: Error executing adbExec. Original error: &apos;Command &apos;&apos;C:\Users\moham\AppData\Local\Android\Sdk\platform-tools\adb.exe&apos; -P 5037 -s emulator-5554 shell &apos;settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy&apos;&apos; exited with code 20&apos;; Command output: cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings

    at getResponseForW3CError (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.ts:1103:34)
    at asyncHandler (C:\Users\moham\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.ts:507:57)
The above error is caused by
Error: Error executing adbExec. Original error: &apos;Command &apos;&apos;C:\Users\moham\AppData\Local\Android\Sdk\platform-tools\adb.exe&apos; -P 5037 -s emulator-5554 shell &apos;settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy&apos;&apos; exited with code 20&apos;; Command output: cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings
cmd: Can&apos;t find service: settings

    at ChildProcess.&lt;anonymous&gt; (C:\Users\moham\.appium\node_modules\appium-uiautomator2-driver\node_modules\teen_process\lib\exec.js:110:19)
    at ChildProcess.emit (node:events:519:28)
    at maybeClose (node:internal/child_process:1101:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:304:5)
    at Process.callbackTrampoline (node:internal/async_hooks:130:17)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:215)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:1034)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:147)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:588)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:574)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:687)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	... 40 more
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
</body>
</html>
